tests/unit/clustering/RandomInitialization_unittest.cc

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
This is a unit test for KMeans, and not for random initialisation. The file should be named accordingly